apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java

```diff
@@ -131,6 +131,21 @@ public void loadSyntheticData() {
     loadSample(dataSource, StaticRifResourceGroup.SYNTHETIC_DATA);
   }
 
+  /**
+   * Runs {@link gov.cms.bfd.pipeline.rif.load.RifLoader} against the {@link
+   * StaticRifResourceGroup#LOCAL_SYNTHETIC} data.
+   */
+  @Test
+  public void loadLocalSyntheticData() {
```

Comment:
Should this have an @Ignore attribute? Do we want to run this on every test pass?

Reply:
Yup good idea. Is there some better way to do this kind of task aside from a test? Does maven (or java in general) have a best practice for utility type commands?

Reply:
Not sure about what is the practice, but I ignore is used in this file for this load utilities. Personally, I would love to have a developer command-line utility for this project.